test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -328,10 +330,34 @@ public void TestListWithNullItem()
             {
                 Assert.Equal(ErrorLevel.Info, error.Level);
                 Assert.Equal("null-value", error.Code);
```

Comment:
can we have a Theory based test for both JSON and YAML? It should cover both value/array/and object